pkg/operator/ceph/csi/template/nfs/csi-nfsplugin-provisioner-dep.yaml

```diff
@@ -21,6 +21,46 @@ spec:
       priorityClassName: {{ .ProvisionerPriorityClassName }}
       {{ end }}
       containers:
+        {{ if .EnableNFSSnapshotter }}
+        - name: csi-snapshotter
+          image:  {{ .SnapshotterImage }}
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--v={{ .SidecarLogLevel }}"
+            - "--timeout={{ .GRPCTimeout }}"
+            - "--leader-election=true"
+            - "--leader-election-namespace={{ .Namespace }}"
+            - "--leader-election-lease-duration=137s"
```

Comment:
How are these leader election timeouts determined? They're the same settings as the other sidecars, right?

Reply:
> How are these leader election timeouts determined? 

They were last changed in this commit 
https://github.com/rook/rook/commit/5e92a30a7782669e4907f5635efac5de8a8efdf6

>They're the same settings as the other sidecars, right?

Yes they are exactly the same, 
https://github.com/rook/rook/blob/9c4592720c924fa6ad4cfcc31332637c7a8da069/pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml#L44